### PR TITLE
[enhancement](nereids) Update stats table config

### PR DESCRIPTION
**Pull request by Kikyou1997**
## Proposed changes

1. set replica count fot stats tbl as :"Math.max(Config.statistic_internal_table_replica_num,Config.min_replication_num_per_tablet)"
2. update comment for stats tbl remove symbol '''

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Kikyou1997** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 45.78 seconds
 stream load tsv:          503 seconds loaded 74807831229 Bytes, about 141 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          29.2 seconds inserted 10000000 Rows, about 342K ops/s
 storage size: 17161724498 Bytes

**Comment by Kikyou1997** [comment]
2.0.0 required

File: fe/fe-core/src/main/java/org/apache/doris/catalog/InternalSchemaInitializer.java [hunk 1]
```diff
@@ -121,12 +121,13 @@ public CreateTableStmt buildAnalysisTblStmt() throws UserException {
...
             {
-                put("replication_num", String.valueOf(Config.statistic_internal_table_replica_num));
+                put("replication_num", String.valueOf(
+                        Math.max(Config.statistic_internal_table_replica_num, Config.min_replication_num_per_tablet)));
             }
         };
         CreateTableStmt createTableStmt = new CreateTableStmt(true, false,
                 tableName, columnDefs, engineName, keysDesc, null, distributionDesc,
-                properties, null, "Doris internal statistics table, don't modify it", null);
+                properties, null, "Doris internal statistics table, DON NOT MODIFY IT", null);
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/InternalSchemaInitializer.java:130]: typo error

**Comment by Kikyou1997** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 45.49 seconds
 stream load tsv:          504 seconds loaded 74807831229 Bytes, about 141 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          67 seconds loaded 1101869774 Bytes, about 15 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.2 seconds inserted 10000000 Rows, about 342K ops/s
 storage size: 17162646680 Bytes

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by morrySnow — APPROVED** [review]

**Pull request merged**